lib/vendors/axis-communications/sv_vendor_axis_communications.c

```diff
@@ -69,6 +71,54 @@ typedef struct _sv_vendor_axis_communications_t {
   sv_vendor_axis_supplemental_authenticity_t supplemental_authenticity;
 } sv_vendor_axis_communications_t;
 
+// Declarations of static functions.
+static svi_rc
+verify_certificate_chain(X509 *trusted_ca, STACK_OF(X509) * untrusted_certificates);
+
+// Definitions of static functions.
+
+/* Verifies the untrusted certificate chain.
+ *
+ * Uses the |trusted_ca| to verify the first certificate in the chain. The last certificate verified
+ * is the |attestation_certificate| which will be used to verify the transmitted public key. */
+static ATTR_UNUSED svi_rc
```

Comment:
Remove "ATTR_UNUSED" ?

Reply:
Since the function is not used yet we will get a compilation warning (treated as error) otherwise. It will be removed when function is used in a later commit.

Reply:
Oki doki.